recipes/spirv-tools/all/conanfile.py

```diff
@@ -44,10 +44,15 @@ def configure(self):
             tools.check_min_cppstd(self, 11)
 
     def requirements(self):
-        self.requires("spirv-headers/{}".format(self._get_compatible_spirv_headers_version()))
+        self.requires("spirv-headers/{}".format(self._get_compatible_spirv_headers_version))
 
+    @property
     def _get_compatible_spirv_headers_version(self):
-        return "1.5.3" if tools.Version(self.version[1:]) >= "2020.2" else "1.5.1"
+        return {
+            "v2019.2": "1.5.1",
+            "v2020.3": "1.5.3",
+            "v2020.5": "1.5.4",
+        }.get(str(self.version))
```

Comment:
This can return `None`.
Maybe throw an exception when none of found?
Right now, conan would print `cannot find 'spirv-headers/None'`

Reply:
fixed